### DISCUSSION TITLE
Garmin Venu - Sensor

### DISCUSSION THREAD

**Discussion by kenlpeters** [cagnulein/qdomyos-zwift#218] [Q&A]
Is it possible to see the qDomyos software as a power, speed, and cadence sensor on my Garmin Venu?

**Comment by cagnulein** [comment]
with ANT+ yes. Did you try?

**Reply by kenlpeters** [reply]
My IOS device does not have Ant+ so it all must done with Bluetooth

**Reply by cagnulein** [reply]
nope so, i guess garmin has some limits reading these datas from something that isn't a real device itself. I tried on my Fenix 5s, it will see the sensors but it won't connect at all. I will give it another shot in the future, i just need time :D
are you a developer? you can help me if you want. I could address you

**Reply by kenlpeters** [reply]
I am/was a developer.  Don't code much these days.  I will try it and let you know as well.  I figured I would ask before trying it.

**Reply by cagnulein** [reply]
you should enable the battery service setting in order to see it in the garmin sensors list. tell me what you will discover! thanks!

**Reply by kenlpeters** [reply]
battery service setting?

**Reply by cagnulein** [reply]
yes in the QZ app. if you don't find it it's because i removed it in the production version. if you want to try it out, i can add you into the beta group. just send me your icloud email to roberto.viola83@gmail.com

**Reply by kenlpeters** [reply]
I send you an email with my icloud account information.

**Comment by cagnulein** [comment]
Done

Il gio 25 mar 2021, 16:47 kenlpeters ***@***.***> ha scritto:

**Reply by kenlpeters** [reply]
So I checked my Garmin Watch at lunch.  It was not able to see the QD software as a sensor.  I checked my peloton app and it worked properly and was able to see it.

**Reply by kenlpeters** [reply]
Also the application won't install through Test Flight.  I removed the production version and it still won't install.  I press install.  It looks like it going to install and then button changes back to "Install"

**Reply by kenlpeters** [reply]
Never mind the install issue I need to turn on VPN... Testflight is blocked at work.

**Comment by cagnulein** [comment]
Ok so try with the battery service

Il gio 25 mar 2021, 18:01 kenlpeters ***@***.***> ha scritto:

**Reply by kenlpeters** [reply]
I turned it on, how do I test it?

**Reply by kenlpeters** [reply]
I am work now.  I will test it with the bike this evening and let you know.

**Comment by cagnulein** [comment]
just connect QZ to your bike and you should see the sensors on the garmin.

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 25 mar 2021 alle ore 19:16 kenlpeters <
***@***.***> ha scritto:

**Reply by kenlpeters** [reply]
I tried this morning and my garmin did not see if the QZ.  I was kinda in a rush to get my workout in so I will fiddle with it this weekend and see if I have success. With that being said.  The Peloton App on my Iphone found the QZ cadence on my iPad without issue.  Now if I can get Peloton to add power and speed to their app I will be golden.

**Comment by cagnulein** [comment]
i guess you should run QZ app on the right side of the screen if you want
just one device. Of course you will lose the ability to send data to
peloton app.
Garmin thing it's something requires some time to setup, so please take
your time to check it :) It's an hard friend :D
R.

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 26 mar 2021 alle ore 12:54 kenlpeters <
***@***.***> ha scritto:

**Reply by kenlpeters** [reply]
Ok we are having a Peloton Tread + delivered tomorrow so I expect I will be in the gym for a while.  I will give a whirl then.

**Comment by cagnulein** [comment]
Oh wow! Congratulations! :)

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 26 mar 2021 alle ore 13:28 kenlpeters <
***@***.***> ha scritto:
